src/element.js

```diff
@@ -36,7 +36,21 @@ function findElement(sendSessionCommand, strategy, selector) {
             getText: getText.bind(null, sendElementCommand),
             getCss: getCss.bind(null, sendElementCommand)
         };
-    });
+    }).catch(() => undefined);
+}
+
+async function findElement(sendSessionCommand, strategy, selector, maxWaitTime = 0) {
```

Comment:
The problem with adding the wait functionality to findElement will require to add it to every other function as well to keep the api consistent. Also I think that waiting for certain condition is not related to webdriver. It is a general utility function. I cound find such packages on npm as well. I would like this package to know only what webdriver api knows. The only difference should to have async function calls with proper error handling instead of REST requests. The main purpose to use this package instead of selenium-webriver is the ability to have a raw native interface without any additional abstractions